src/components/application_manager/include/application_manager/application_data_impl.h

```diff
@@ -43,6 +45,7 @@
 namespace application_manager {
 
 namespace mobile_api = mobile_apis;
+typedef std::map<std::uint32_t, bool> ChoiceSetAllowedMap;
```

Comment:
@IGapchuk you don't need to specify std for uint32_t

Reply:
@AKalinich-Luxoft Fixed in fbc582d9bf3feaa96a345733d977106eea7a02b4.